MyLinkedListTest.java

```diff
@@ -0,0 +1,79 @@
+package pavelGo;
+
+import org.junit.*;
+import org.junit.rules.Timeout;
+
+public class MyLinkedListTest extends Assert {
+
+    @Rule
+    public final Timeout timeout = new Timeout(1000);

```

Comment:
why you need this ?

Reply:
limiting the execution time of the program, is it better to remove it?